### PR TITLE
Add docs to help users diagnose issues before creating them

### PR DESCRIPTION
**Pull request by jasongrout**
## Code changes

Adds documentation to help users diagnose issues and create relevant and complete issues.

## Backwards-incompatible changes

None

## TODO

- [x] Update the issue templates

### CODE REVIEW

**Comment by jasongrout** [comment]
I think this is ready for feedback, except for pipenv instructions for creating a new fresh environment. Anyone know how to create a new environment in pipenv with a fresh JupyterLab install? @pivanov?

File: docs/source/getting_started/issue.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+     2. Activate the environment::
+       
+         conda activate jlab-test
+
+     3. Start JupyterLab::
+
+         jupyter lab
+
+   If you are using pipenv: (someone please update this area with instructions for creating a new environment in pipenv)
```
Comment (blink1073) [inline docs/source/getting_started/issue.rst]: Ha!  I have not used pipenv.

**Comment by jasongrout** [comment]
> Should we also mention checking for security software being enabled or trying a different browser to single out a browser compatibility issue?

Good point.

I think I'm going to take out the 'disable browser extensions' - that has only rarely come up, and we can diagnose that on an issue. This is meant to be a filter to quickly catch the majority of issues that should be opened elsewhere.

**Comment by jasongrout** [comment]
> Should we also mention rying a different browser to single out a browser compatibility issue?

Done!

**Comment by jasongrout** [comment]
> checking for security software being enabled

I think this is rare enough we can diagnose that in an issue? Has it come up that often?

**Comment by blink1073** [comment]
I've seen it manifest in accessing URLs, files, and the page itself.

**Comment by blink1073** [comment]
Including weird stuff like a file with the name `ad` in it.

**Comment by blink1073** [comment]
Also websocket security settings

**Comment by jasongrout** [comment]
Done. Feel free to add to the short checklist I just added at the end of the diagnosis section.

**Comment by blink1073** [comment]
Found a pipenv reference  :wink:

https://github.com/jupyterlab/jupyterlab/blob/1593760ac731234e48ba9c3701bb8dea1d5861ae/docs/source/getting_started/installation.rst#pipenv

**Comment by jasongrout** [comment]
I spent a few minutes reading up on pipenv, and it still wasn't clear to me how to create a fresh new environment with nothing by jlab installed. I'll ask @pivanov :).

**Comment by blink1073** [comment]
Looks good, thanks!

**Comment by blink1073** [comment]
@meeseeksdev backport to 1.0.x

**Pull request merged**